### ISSUE TITLE
[Latest release not published into Debian repository] (Version: [1.9])

### ISSUE THREAD

**Issue by the-smooth-operator** [ripple/rippled#4154]
## Issue Description

The last rippled release is not yet in the official Debian Repositories: https://repos.ripple.com/repos/rippled-deb

## Steps to Reproduce

```
apt-get update && apt-cache show rippled |head -2
Package: rippled
Version: 1.8.5-1
```

## Expected Result

```
apt-get update && apt-cache show rippled |head -2
Package: rippled
Version: 1.9.0-1
```

## Supporting Files

Are you planning on maintaining this repo up to date with latest versions? It's ok if not, we just want to plan accordingly so we are running the latest version.

**Comment by muratso** [comment]
I have the same question. In fact, I can't access this repository from the browser anymore (which I used to be able to).

**Comment by gitmewai** [comment]
Yes same here too. It looks like the `1.9.0-1` is shown under `unstable` suite only

**Comment by danny-goncalves-anchor** [comment]
From what I saw, that version isn't fully released yet, the amendment for NFTs is still under voting (and not approved) https://xrpscan.com/amendments.

**Comment by the-smooth-operator** [comment]
in that case, should it be defined as Release Candidate? Or any other naming convention.
Looking at the official Releases page, it seems like it is stable/prod-ready.
https://github.com/ripple/rippled/releases/tag/1.9.0

**Comment by nbougalis** [comment]
Packages for the **1.9.0** release were held back after several third party server operators identified issues that manifested, primarily, as poor syncing performance. 

Those issues were addressed with the **1.9.1** release (with commits 245174c42c87bd24404bbe16d4e56ee0b7f8fb4c and b68a66928c7b6d3b9040980bfa793509a72f7cc5).

Packages for **1.9.1**, which was finalized on May 22, 2022 should be available. 

I am going to close this issue, but if you have issues upgrading to **1.9.1** please feel free to open a new issue.
